experiment_1/004_results_data_template/mapping.csv

```diff
@@ -8,35 +8,35 @@ Transmission_Flows_Electricity_Grid,GWh,electricity-transmission,electricity,lin
+EnergyConversion_FuelCosts_Electricity_Nuclear_ST,Eur,uranium-nuclear-st,uranium,conversion,nuclear-st,cost_carrier
 EnergyConversion_Invest_Electricity_CH4_GT,Eur,ch4-gt,ch4,conversion,gt,cost_invest
-EnergyConversion_Invest_Electricity_Nuclear_ST,Eur,uranium-nuclear,uranium,dispatchable,nuclear,cost_invest
+EnergyConversion_Invest_Electricity_Nuclear_ST,Eur,uranium-nuclear-st,uranium,conversion,nuclear-st,cost_invest
 EnergyConversion_SecondaryEnergy_Electricity_CH4_GT,GWh,ch4-gt,ch4,conversion,gt,flow_out
-EnergyConversion_SecondaryEnergy_Electricity_Nuclear_ST,GWh,uranium-nuclear,uranium,dispatchable,nuclear,flow_out
+EnergyConversion_SecondaryEnergy_Electricity_Nuclear_ST,GWh,uranium-nuclear-st,uranium,conversion,nuclear-st,flow_out
 EnergyConversion_VarOM_Electricity_CH4_GT,Eur,ch4-gt,ch4,conversion,gt,cost_varom
-EnergyConversion_VarOM_Electricity_Nuclear_ST,Eur,uranium-nuclear,uranium,dispatchable,nuclear,cost_varom
-Storage_Capacity_Electricity_H2CavernCharge,MW,h2-storage,h2,storage,storage,capacity
-Storage_Capacity_Electricity_H2CavernDischarge,MW,h2-storage,h2,storage,storage,capacity
-Storage_Capacity_Electricity_H2CavernStorage,MW,h2-storage,h2,storage,storage,storage_capacity
-Storage_Capacity_Electricity_LiIonBatteryCharge,MW,electricity-battery,electricity,storage,battery,capacity
-Storage_Capacity_Electricity_LiIonBatteryDischarge,MW,electricity-battery,electricity,storage,battery,capacity
-Storage_Capacity_Electricity_LiIonBatteryStorage,MW,electricity-battery,electricity,storage,battery,storage_capacity
-Storage_FixOM_Electricity_H2CavernStorage,Eur,h2-storage,h2,storage,storage,cost_fixom
-Storage_FixOM_Electricity_LiIonBatteryStorage,Eur,electricity-battery,electricity,storage,battery,cost_fixom
-Storage_Input_Electricity_H2CavernStorage,GWh,h2-storage,h2,storage,storage,flow_in
-Storage_Input_Electricity_LiIonBatteryStorage,GWh,electricity-battery,electricity,storage,battery,flow_in
-Storage_Invest_Electricity_H2CavernStorage,Eur,h2-storage,h2,storage,storage,cost_invest
-Storage_Invest_Electricity_LiIonBatteryStorage,Eur,electricity-battery,electricity,storage,battery,cost_invest
-Storage_Losses_Electricity_H2CavernStorage,GWh,h2-storage,h2,storage,storage,loss
-Storage_Losses_Electricity_LiIonBatteryStorage,GWh,electricity-battery,electricity,storage,battery,loss
-Storage_Output_Electricity_H2CavernStorage,GWh,h2-storage,h2,storage,storage,flow_out
-Storage_Output_Electricity_LiIonBatteryStorage,GWh,electricity-battery,electricity,storage,battery,flow_out
-Storage_VarOM_Electricity_H2CavernStorage,Eur,h2-storage,h2,storage,storage,cost_varom
-Storage_VarOM_Electricity_LiIonBatteryStorage,Eur,electricity-battery,electricity,storage,battery,cost_varom
+EnergyConversion_VarOM_Electricity_Nuclear_ST,Eur,uranium-nuclear-st,uranium,conversion,nuclear-st,cost_varom
+Storage_Capacity_Electricity_H2CavernCharge,MW,h2-storage,electricity,asymmetric storage,h2_cavern,capacity_charge_sum
```

Comment:
It should be asymmetric_storage with an underscore.

Reply:
Without in order to be consistent with other names in TYPEMAP

Reply:
Can stay like this to keep consistency with oemof.tabular